test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -219,6 +225,33 @@ func clearEnv() {
 	os.Unsetenv(latencyTestCpus)
 }
 
+// checkHyperthreading determines if the BZ 2094046 test should run based on HT status.
+// Returns true if the test should run (HT enabled), false if it should be skipped (HT disabled).
+func checkHyperthreading() (bool, error) {
+	profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+	if err != nil {
+		return false, fmt.Errorf("get performance profile: %w", err)
+	}
+
+	workerNodes, err := nodes.GetByLabels(testutils.NodeSelectorLabels)
+	if err != nil {
+		return false, fmt.Errorf("get worker nodes: %w", err)
+	}
+	workerNode := &workerNodes[0]
+
+	ctx := context.Background()
+	set, err := cpuset.Parse(string(*profile.Spec.CPU.Isolated))
+	if err != nil || set.Size() == 0 {
+		return false, fmt.Errorf("failed to parse isolated CPUs from profile")
+	}
+	cpuID := set.List()[0]
+
+	isHTEnabled := nodes.IsHyperthreadingEnabled(ctx, cpuID, workerNode)
```

Comment:
I'd check the node settings (possibly `/proc/cmdline`) or actually any random CPU. To put it differently, why the first isolated CPU is significant and why is it better than, say, cpu#0 ?